editor/frontend/src/ElementGeometryTool.ts

```diff
@@ -204,39 +206,44 @@ export abstract class ElementGeometryCacheTool extends ElementSetTool implements
     this.agenda.elements.forEach((id) => { overrides.overrideElement(id, app); });
   }
 
-  protected updateAgendaAppearanceProvider(): void {
+  protected updateAgendaAppearanceProvider(drop?: true | undefined): void {

```

Comment:
You ran lint, right? I might have expected it to complain about this.

Reply:
Yeah, no errors, but it's changed now.